docs/UserGuide.md

```diff
@@ -45,106 +47,170 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
-*  `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
+*  `project -e 1 n/ProjectX` Edits the project with `PROJECT_ID` 1 to be renamed to ProjectX.
+*  `project -e 2 r/AgentY/ProjectY` Edits the project with `PROJECT_ID` 2 to be associated with new repository link.
+*  `project -e 3 n/ProjectZ r/AgentZ/ProjectZ` Edits the project with `PROJECT_ID` 3.
+
+### Deleting a project : `project -d`
+
+Deletes a project from the application.
+
+Format: `project -d PROJECT_ID`
+
+* Deletes the project with the specified `PROJECT_ID`.
+* The ID **must be a positive integer** 1, 2, 3, …​
+
+Examples:
+* `project -d 1` Deletes project with `PROJECT_ID` 1.
+
+### Tagging a client : `client -t`
+
+Tags a client to a project.
+
+Format: `client -t PROJECT_ID n/CLIENT_NAME [p/CLIENT_CONTACT_NUMBER] [e/CLIENT_EMAIL]…​`
+
+* Adds the client to the project with the specified `PROJECT_ID`.
+
+Examples:
+*  `client -t 1 n/Amy` Tags the project with `PROJECT_ID` 1 with a client with `CLIENT_NAME` Amy.
+*  `client -t 2 n/Bob p/91234567` Tags the project with `PROJECT_ID` 2 with a client with `CLIENT_NAME` Bob and 
+   `CLIENT_CONTACT_NUMBER` 91234567.
+*  `client -t 3 n/Charlie e/charlie@gmail.com` Tags the project with `PROJECT_ID` 2 with a client with `CLIENT_NAME` 
+    Charlie and `CLIENT_EMAIL` charlie@gmail.com.
+*  `client -t 2 n/Dave p/91111111 e/dave@gmail.com` 
 
-### Locating persons by name: `find`
+### Editing a client : `client -d`
 
-Finds persons whose names contain any of the given keywords.
+Edits the client for a specific project.
 
-Format: `find KEYWORD [MORE_KEYWORDS]`
+Format: `client -e i/PROJECT_ID n/CLIENT_NAME [p/CLIENT_CONTACT_NUMBER] [e/CLIENT_EMAIL]…​`
 
-* The search is case-insensitive. e.g `hans` will match `Hans`
-* The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-* Only the name is searched.
-* Only full words will be matched e.g. `Han` will not match `Hans`
-* Persons matching at least one keyword will be returned (i.e. `OR` search).
-  e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
+* Edits the client of the project with the specified `PROJECT_ID`.
+* At least one of the optional fields must be provided.
+* Existing values will be updated to the input values.
 
 Examples:
-* `find John` returns `john` and `John Doe`
-* `find alex david` returns `Alex Yeoh`, `David Li`<br>
-  ![result for 'find alex david'](images/findAlexDavidResult.png)
+*  `client -e 1 n/Amy` Edits the client tagged to project with `PROJECT_ID` 1 with new `CLIENT_NAME` Amy.
+*  `client -e 2 n/Bob p/91234567` Edits the client tagged to project with `PROJECT_ID` 2 with new `CLIENT_NAME` Amy 
+   and `CLIENT_CONTACT_NUMBER` 91234567.
+*  `client -e 3 n/Charlie e/charlie@gmail.com` Edits the client tagged to project with `PROJECT_ID` 3 with new 
+   `CLIENT_NAME` Charlie and `CLIENT_EMAIL` charlie@gmail.com.
+*  `client -e 2 n/Dave p/91111111 e/dave@gmail.com`
```

Comment:
I think u missed out the explanation for this line. 
```suggestion
*  `client -e 2 n/Dave p/91111111 e/dave@gmail.com` Edits the client tagged to project with `PROJECT_ID` 2 with new `CLIENT_NAME` Dave, `CLIENT_CONTACT_NUMBER` 91111111 and `CLIENT_EMAIL` dave@gmail.com.
```

Reply:
I had left the explanation out as it seemed redundant given the previous two commands, but I can include it in a new commit for better consistency...